tests/extmod/btree1.py

```diff
@@ -56,6 +60,18 @@
 for k, v in db.items(None, None, btree.DESC):
     print((k, v))
 
+print("---")
+# R_CURSOR flag
+print(db.seq(1, b"foo1"))
+print(db.seq(1, b"qux"))
+
+try:
+    db.seq(b"foo1")
+except OSError as e:
+    assert e.args[0] == uerrno.EINVAL
```

Comment:
Better to actually print something rather than assert.  Eg `print(e.args[0] == uerrno.EINVAL)`.  That way, if the test fails here you can diff the output files and see what went wrong.